entity-framework/core/extensions/index.md

```diff
@@ -51,6 +51,14 @@ IWAPI (Instant Web API) is a scaffolding engine for .NET Core that can automate
 
 [Website](https://instantwebapi.com/)
 
+### Moq.EntityFrameworkCore
+
+This library helps you mock EntityFramework contexts. It allows you to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.
+
+For EF Core: 3, 5, 6, 7.
```

Comment:
I meant move this up to the sentence above, som it becomes a single paragraph (just like almost all other extensions on this list)